flutter/example/linux/flutter/generated_plugin_registrant.cc

```diff
@@ -0,0 +1,13 @@
+//
```

Comment:
There are a couple generated files. They aren't gitignored according to the template. I'm not sure yet wether we can ignore them.

Reply:
They can't be ignored